docs/tutorial/1-prerequisites.md

```diff
@@ -27,7 +27,7 @@ On Windows, we've got an install script that will use [Scoop][scoop] if availabl
 iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/tilt-dev/tilt/master/scripts/install.ps1'))
 ```
 
-If you'd rather install manually or with an alternative method, refer to the [Alternative Installations][install-tilt-alternate] guide.
+If you'd rather install manually or another method, refer to the guide on [Alternative Installations][install-tilt-alternate].
```

Comment:
This is extremely minor and possibly idiosyncratic, but I think it's nicer when a link ends a sentence without a dangling non-link word after it.